memo-parser/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
let's make these gitignored and copied in via prepublish from the root, rather than duplicating the content.

Reply:
or better yet, what about making this a proper monorepo a use lerna@3 to deal with this.

Reply:
absolutely not; for one, we'd have to move all of our packages into some kind of "packages" dir, which is too much git churn; but also in my experience lerna simply does not work well beyond `bootstrap` and `exec`, and i have no interest in using it on additional projects.